pkg/reconcile/pipeline/infinispan/handler/provision/gossiprouter.go

```diff
@@ -68,6 +68,7 @@ func GossipRouter(i *ispnv1.Infinispan, ctx pipeline.Context) {
 		args := []string{
 			"-port", strconv.Itoa(consts.CrossSitePort),
 			"-dump_msgs", "registration",
+			"-suspect", strconv.FormatBool(*i.Spec.Service.Sites.Local.Discovery.SuspectEvents),
```

Comment:
This is safe if we don't use a pointer, however if `SuspectEvents` is nil after an upgrade then we will have issues.